File: help/eng_gb/exercicios_da_ficha_7/Ficha07_004/Ficha07_004.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Summing Pairs [0,10]\n");
    int sum = 0 ;
    int i = 0 ;
    for( int i = 1; i < 6; i = i + 1 ) {
        int f = 2 * i ;
        printf("%d + %d",sum ,f );
        sum +=f;
        printf(" = %d\n",sum );
    }
    printf("\nSum = %d",sum );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:36

File: help/eng_gb/exercicios_da_ficha_7/Ficha07_004/Ficha07_004.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha07_004 {

    //Main Function
    static void Main() {
        Console.Write("Summing Pairs [0,10]\n" );
        int sum = 0 ;
        for( int i = 1; i < 6; i = i + 1 ) {
            int f = 2 * i ;
            Console.Write(sum + " + " + f );
            sum +=f;
            Console.Write(" = " + sum + "\n" );
        }
        Console.Write("\nSum = " + sum );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:36
